### PR TITLE
Enhanced the ui of mobile screen

### LINKED ISSUE #55: [Feature Request]: improve the layout of hero section for smaller screen
### Is your feature request related to a problem? Please describe.

....

### Describe the solution you'd like

.....

### Describe alternatives you've considered

_No response_

### Developer Help

image

### PR DESCRIPTION
**Pull request by abansal15**
## Overview

1. This PR fixes or fixes part of #55 
2. This PR does the following: [Explain here what your PR does and why]

## Essential Checklist

- [x] The PR title starts with "Fix #55: "
- [x] "Allow edits from maintainers" is checked. (See [here](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) for instructions on how to enable it.)
- [x] The PR is made from a branch that's **not** called "main/master".

## PR Pointers

- If you need a review or an answer to a question, and don't have permissions to assign people, **leave a comment** like the following: "{{Question/comment}} @{{reviewer_username}} PTAL".
- Never force push. If you do, your PR will be closed.

### CODE REVIEW

**Comment by sambhavgupta0705** [comment]
Please check all the tasks and the issue linked to this PR

**Comment by abansal15** [comment]
@sambhavgupta0705 i dont understand what i have to do is you are telling to tick the essential checklist items i have done it or anything else ?

**Comment by sambhavgupta0705** [comment]
I am talking about this

**Comment by abansal15** [comment]
@sambhavgupta0705 i have done it

**Comment by abansal15** [comment]
image

File: src/components/HeroSection/index.jsx [hunk 1]
```diff
@@ -5,11 +5,13 @@ const HeroSection = () => {
...
     <main>
-      <div className="h-[15vh]" />
+      <div className="h-[0vh]" />
       <div className={classNames({
-        'h-[75vh] w-screen relative': true,
+        'h-[40vh] w-screen relative': true,
         'flex items-center justify-between overflow-hidden': true,
       })}>
+
+        <img src={"https://images.pexels.com/photos/6077089/pexels-photo-6077089.jpeg"} alt={"blueBlob"} className="mobile-sm:!scale-[1.5] hero-section-sm:block hidden absolute mobile:scale-[1.25] w-full rotate-[0deg] left-[0rem] z-0" />
```
Comment (sambhavgupta0705) [inline src/components/HeroSection/index.jsx:14]: change the `alt`

File: tailwind.config.js [hunk 2]
```diff
@@ -14,7 +14,7 @@ module.exports = {
           max: '400px',
         },
         mobile: {
-          max: '531px',
+          max: '470px',
```
Comment (sambhavgupta0705) [inline tailwind.config.js:17]: Don't change this it will affect other components also

**Review by sambhavgupta0705 — CHANGES_REQUESTED** [review]

**Comment by abansal15** [comment]
@sambhavgupta0705 Done the required changes please review it..

**Review by sambhavgupta0705 — APPROVED** [review]

**Pull request merged**